.github/workflows/tests.yml

```diff
@@ -0,0 +1,111 @@
+---
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django: [2.2.*, 3.0.*, 3.1.*, 3.2b1]
```

Comment:
Using `2b1` here isn't ideal because we need to update it once 3.2.0 has been released.  But I'd say it's good enough for the initial version and important enough due to 3.2 being a LTS release.